pkgs/top-level/python-packages.nix

```diff
@@ -8131,6 +8176,30 @@ in modules // {
     };
   };
 
+  django_raster = buildPythonPackage rec {
+    name = "django-raster-${version}";
+    version = "0.1.7a1";
+
+    # Test data is missing
+    doCheck = false;
+
+    src = pkgs.fetchFromGitHub {
+      sha256 = "1ssvs3yf3wsk0yjsbzg9bla6kfr8dqv3hxbm2g4f3bfkxp4kj5gq";
+      rev = "011cd72d1308164b133db72123cdd2cd86fd9ed6";
+      repo = "django-raster";
+      owner = "geodesign";
+    };
+
+    buildInputs = with self ; [ django_1_9 ];
```

Comment:
I did put it here to allow the django extension to be used with various django versions (even if this one is not such a good example since it requires .django >= 1.9`).

I am not quite sure what should be the preferred approach to get the various extensions we have available to the various django versions. I saw a PR removing an override based approach recently. What I did here is a kind of a “poor man's” workaround.

If you prefer this extension to force one and only one django version, I’m ok to to put django in the propagated build inputs.